src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -1298,6 +1329,11 @@ protected Object readResolve() {
             maxTotalUses = -1;
         }
 
+         // 1.45 new parameters
```

Comment:
The version number in the comment is out of date. Maybe the just remove the comment because we cannot know when the PR will be released?

Reply:
Removed!